src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -19,39 +35,67 @@ export default function EligibilityForm() {
       });
     }
   };
+
   return (
-    <View style={styles.container}>
```

Comment:
I'd recommend keeping an outer container for the ScrollView. This can set the background color and keep everything centered using flex properties.